Bottom: 800184550585eaddea8712a452f5ba4292a74bfd
Top:    566b05f65574bccb10d49e142ea0082a0826bc7e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 20:31:12 +0000

Turn on thindraw poly eye-candy


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 3b54c6e..8229e63 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -904,7 +904,7 @@ ghid_thindraw_pcb_polygon (hidGC gc, PolygonType *poly, const BoxType *clip_box)
 
   double old_alpha_mult = gtk_gc->alpha_mult;
   common_thindraw_pcb_polygon (gc, poly, clip_box);
-#if 0
+#if 1
   ghid_set_alpha_mult (gc, gtk_gc->alpha_mult * 0.25);
   hid_draw_fill_pcb_polygon (gc, poly, clip_box);
   ghid_set_alpha_mult (gc, old_alpha_mult);
